src/javascript/lib/landing_pages/ico.js

```diff
@@ -124,6 +125,106 @@ function clearHash() {
     }
 }
 
+function signUpInit() {
+    var el_email   = document.getElementById('email');
+    var el_signup  = document.getElementById('signup');
+    var el_success = document.getElementById('success');
+
+    var ws = wsConnect();
+
+    function sendVerifyEmail() {
+        var trimmed_email = el_email.value.trim();
+        wsSend(ws, {
+            verify_email: trimmed_email,
+            type        : 'account_opening'
+        });
+    }
+
+    var validation_set = false; // To prevent validating before submit
+
+    document.getElementById('frm_verify_email').addEventListener('submit', function (evt) {
+        evt.preventDefault();
+
+        ws = wsConnect();
+
+        if (!validateEmail(el_email.value)) {
+            if (!validation_set) {
+                ['input', 'change'].forEach(function (evt) {
+                    el_email.addEventListener(evt, function () {
+                        setValidationStyle(el_email, !validateEmail(el_email.value));
+                    });
+                });
+                setValidationStyle(el_email, !validateEmail(el_email.value));
+                validation_set = true;
+            }
+            return false;
+        }
+
+        if (ws.readyState === 1) {
+            sendVerifyEmail();
+        } else {
+            ws.onopen = sendVerifyEmail;
+            ws.onmessage = function(msg) {
+                var response = JSON.parse(msg.data);
+                setValidationStyle(el_email, response.error);
+                if (!response.error) {
+                    el_signup.classList.add('invisible');
+                    el_success.classList.remove('invisible');
+                }
+            }
+        }
+    });
+
+    ws.onmessage = function(msg) {
+        var response = JSON.parse(msg.data);
+        setValidationStyle(el_email, response.error);
+        if (!response.error) {
+            el_signup.classList.add('invisible');
+            el_success.classList.remove('invisible');
+        }
+    };
+
+    ws.onclose = function(e) {
+        setTimeout(function() {
+            // console.log('websocket connection dropped. connect in 1 seconds');
+            ws = wsConnect();
+        }, 1000);
+    };
+
+    ws.onerror = function(err) {
+        // console.error('Socket encountered error: ', err.message, 'Closing socket');
+        ws.close();
+    };
+
+    // Store gclid
+    var gclid = getParamValue(document.referrer, 'gclid');
+    if (gclid) {
+        localStorage.setItem('gclid', gclid);
+    }
+}
+
+function validateEmail(email) {
+    return /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,63}$/.test(email);
+}
+
+function setValidationStyle(element, has_error) {
+    var error_class = 'error-field';
+    var invisible_class = 'invisible';
+    element.classList[has_error ? 'add' : 'remove'](error_class);
```

Comment:
@zakibinary Please use `element.classList.toggle()` since we are targeting `IE9+`.